careers/careers/templates/careers/listings.jinja

```diff
@@ -57,7 +57,7 @@
           data-type="{{ position.position_type }},"
           data-location="{{ position.location }},">
         <td class="title"><a href="{{ position.get_absolute_url() }}">{{ position.title }}</a></td>
-        <td class="location">{{ position.location_list|join(", ") }}</td>
+        <td class="location">{{ position.location }}</td>
```

Comment:
I don't think this will change much. You can see in the `models.py` file that the `location` field and the `location_list` field get their data from the same place. The `location_list` is just the data in `location` split into an array on commas. I'm guessing fixing this issue is going to require a change to the source of the data in the location field of the DB.

Reply:
Old way:
![image](https://user-images.githubusercontent.com/10035299/113165498-ce0ef080-91f6-11eb-9aa5-5ff52ce20e9b.png)

New Way:
![image](https://user-images.githubusercontent.com/10035299/113165595-e54dde00-91f6-11eb-899a-eefa552d5cfe.png)